oem/ibm/libpldmresponder/inband_code_update.cpp

```diff
@@ -222,9 +313,13 @@ void CodeUpdate::setVersions()
                 auto imageInterface = "xyz.openbmc_project.Software.Activation";
                 auto imageObjPath = path.str.c_str();
 
-                try
-                {
-                    if (isCodeUpdateInProgress())
```

Comment:
why is this check removed and added inside try? is there need to do a getDBUSVariant call if code update is not in progress?